extra/import_data.py

```diff
@@ -1,197 +1,158 @@
 #!/usr/bin/env python3
 
-"""
-Usage: Import Alissa export in the form of .xsls into a MongoBD database.
-       Make sure the data is in a sub folder called 'input'.
-Python version: 3.7.1
-
-Input .xlsx format
-[Variant List] - spans 1A - 1E
-[General Metadata] -
-    - header spans 2A - 8B
-    - value spans 2C - 8E
-[Annotation Sources]
-    - header spans 9A - 9B
-    - subheader spans 9C - 22C
-    - value spans 9D - 22H
-[Empty line]
-[Data Table of unknown length with first row as header]
-"""
-
 import os
-import io
 import time
-import pymongo
-import warnings
-import progressbar
-import argparse
 import sys
+import pymongo
 import logging
 import logging.config
 import yaml
-from openpyxl import load_workbook
+import config
+import dateutil.parser
+from requests.exceptions import HTTPError
+from alissa_interpret_client.alissa_interpret import AlissaInterpret
+from datetime import datetime, timezone
+
+# Create Alissa connection based on config.py (fill details there)
+client = AlissaInterpret(
+    base_uri=config.alissa_base_uri, client_id=config.alissa_client_id, client_secret=config.alissa_client_secret,
+    username=config.alissa_username, password=config.alissa_password
+)
 
 # configuration for the logging file. This file logs the import of excel files into MongoDB and errors
-with open(os.path.dirname(__file__) + "/logging_config.yml", 'r') as configfile:
+with open("./logging_config.yml", 'r') as configfile:
     logging.config.dictConfig(yaml.safe_load(configfile))
 
 logger = logging.getLogger(__name__)
-logger.debug("File upload script 'import_data.py' started")
-
+logger.info("File upload script 'import_data.py' started")
```

Comment:
This should all be moved to the `main` function.

Reply:
I found that outside of the main it was a lot clearer, because I also use the client in a different function. It makes it better usable outside main (also for future functions if needed)

Reply:
See earlier comment about globals. If you need client in multiple functions it should be added as a parameter of the functions.

Reply:
will make the main according to best practice and not have global variables